### ISSUE TITLE
Login form fill does not work on fortuneo.fr

### ISSUE THREAD

**Issue by nbarraille** [bitwarden/browser#332]
Not filling. Doesn't support field names in French?

**Comment by kspearrin** [comment]
I don't see anything I can do here. It seems this website is explicitly trying to stop us from filling the information.

**Comment by Primokorn** [comment]
Some websites don't want to support password managers :)
They only support SMS as a 2nd factor. Yeah there's nothing to expect from them. Banks are definitively not good examples in terms of security...

**Comment by kevinrambaud** [comment]
I know the original problem regarding the password auto-filling is undoubtedly not happening anymore, but I was having an auto-filling issue with the username not being auto-filled.

For those facing the same issue, I have been able to fix that by adding a [Custom field](https://bitwarden.com/help/custom-fields/) of type "Linked" targeting the "Username" and I named it "LOGIN" all caps (that's the name of the input element corresponding to the username on Fortuneo's authentication page).

Hope it helps those facing this problem.
